### PR TITLE
[Arts 477] Bootstrap user login (display staff profile - front end)

### PR DESCRIPTION
**Pull request by KateSant**
## Description
*  makes asynch request for profile data and displays it on page.
*  new interceptor to add auth token into header of requests to backend services
### Changes
- ARTS-477  subtask ARTS-548

### CODE REVIEW

File: src/app/profile.service.ts [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 import { Observable, of } from 'rxjs';
+import {HttpClient} from "@angular/common/http";
 
 @Injectable({
   providedIn: 'root'
 })
 export class ProfileService {
 
-  constructor() { }
+  private profileUrl = 'http://localhost:8081/practitioner/profile'; // temp. Inject this in task 549
```
Comment (alexbaxter) [inline src/app/profile.service.ts]: This will come from api-endpoint-service once feature/ARTS-549-inject-api-endpoint is merged

**Review by alexbaxter — APPROVED** [review]
Looks OK, not sure I'm competent enough in TypeScript/Angular to judge though.

**Comment by KateSant** [comment]
Discussed the failing cucumber test with @sameerapurini .  This test is looking for the name of the user to be displayed on the page.  When the cucumber test was written, the name information was extracted from the token.   However following this story, the name information comes from a backend service instead.   So the test is still valid but we need to think about the test environment for an end to end service with the backend.   We agreed to leave the for now and revisit it once new /profile service is available.  That should hopefully be PR'd tomorrow - it's caught behind 609.

**Review by shajidissa — APPROVED** [review]

**Review by alexbaxter — APPROVED** [review]
Approved

**Review by alexbaxter — APPROVED** [review]
Approved

**Pull request merged**